dockers/docker-dhcp-server/cli-plugin-tests/test_show_dhcp_server.py

```diff
@@ -0,0 +1,57 @@
+import sys
+from unittest import mock
+
+from click.testing import CliRunner
+
+import utilities_common.cli as clicommon
+
+sys.path.append('../cli/show/plugins/')
+import show_dhcp_server
+
+
+class TestShowDHCPServer(object):
+    def test_plugin_registration(self):
+        cli = mock.MagicMock()
+        show_dhcp_server.register(cli)
+
+    def test_show_dhcp_server_ipv4_lease_without_dhcpintf(self, mock_db):
+        expected_stdout = """\
+Interface            MAC Address        IP             Lease Start    Lease End
+-------------------  -----------------  -----------  -------------  -----------
+Vlan1000|Ethernet10  10:70:fd:b6:13:00  192.168.0.1     1677640581   1677641481
```

Comment:
We need convert the timestamp to datetime format. @yaqiangz for fyi

Reply:
Yes, we need to convert it to datetime

Reply:
Closed